View/media_control_panel.py

```diff
@@ -11,3 +12,18 @@ def __init__(self):
         to the panel using some layout.
         """
         super().__init__()
+        
+        # Create play pause btn
+        self.play_pause_button = QPushButton()
+        self.play_pause_button.setIcon(self.style().standardIcon(QStyle.SP_MediaPlay))
+        
+        #Create horizontal box layout
+        horizontal_layout = QHBoxLayout()
+        horizontal_layout.setContentsMargins(0, 0, 0, 0)
+        horizontal_layout.addWidget(self.play_pause_button)
+
+        #Add vertical box layout to add all widgets
```

Comment:
We do not need a vertical box layout for this widget. Remove lines 25-27, and change the layout set in line 29 to the horizontal_layout instead.